### ISSUE TITLE
training/testing set split for gopro dataset

### ISSUE THREAD

**Issue by XiaoyuShi97** [tarun005/FLAVR#17]
Hi, I download GOPRO_Large_all.zip from https://seungjunnah.github.io/Datasets/gopro.html. I find that there are 33 folders. But I did not see any hint on which folders are of training set or test set. Is this link right?

**Comment by tarun005** [comment]
[GoPro_test.txt](https://github.com/tarun005/FLAVR/files/6255129/GoPro_test.txt)
[GoPro_train.txt](https://github.com/tarun005/FLAVR/files/6255130/GoPro_train.txt)

here are the files for the train and test split used in the paper.

**Comment by XiaoyuShi97** [comment]
Thx!!
